src/components/Card/Card.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+interface CardProps {
+    /** Uh id  */
+    id?: string;
+
+    /** ClassName that appears in addition to "card" */
+    extraClassName?: string[];
+
+    /** Optional modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+    modifiers?: string[];
+
+    /** Optional blockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+    blockName?: string;
+
+    /** Heading that appears within the card but above the three columns within it */
+    heading?: JSX.Element;
+
+    /** Footer that appears within the card but below the three columns within it */
+    footer?: JSX.Element;
+
+    /** First column within the card */
+    image?: JSX.Element;
+
+    /** Main content within the card */
+    content?: JSX.Element;
+
+    /** Calls to Action for the card, such as "download" on a card for a book */
+    ctas?: JSX.Element;
```

Comment:
I think that these should be typed as `React.ReactNode`s so that they can accept anything React would actually accept for rendering purposes. See an explanation [here](https://github.com/typescript-cheatsheets/react-typescript-cheatsheet/blob/cc2ec67e0b36b7e838df8f2a54b4cd84df6c78cd/README.md#useful-react-prop-type-examples).

I generally prefer `React.ReactNode` for anytime I'm passing components around